src/content/advanced/cluster-management/new-cluster-api/index.md

```diff
@@ -0,0 +1,124 @@
+* **Simplified Provider Addition**: CAPI's composable architecture makes it easier to add new infrastructure providers, expanding our offerings beyond AWS, Azure, and our current on-premises solutions.
+* **Multi-Cloud Capabilities**: CAPI supports managing workload clusters from multiple providers under a single management cluster, enabling true multi-cloud deployment scenarios.
+
+## Differences between CAPI and Legacy Cluster Management
+
+The transition to Kubernetes Cluster API (CAPI) introduces several noteworthy differences compared to our legacy cluster management infrastructure. These changes aim to enhance flexibility, expand tooling compatibility, and streamline cluster management operations.
+
+### Cluster CRs Packaged as Helm Charts
+
+CAPI-based cluster configuration is now encapsulated within Helm charts, providing a more controlled and maintainable approach. This approach offers several benefits:
+
+* **Limited Configuration Exposure:** Only tested and stable configuration options are exposed, ensuring a consistent and reliable cluster experience.
+
+* **Ease of Configuration Enabling:** Most CAPI features can be easily enabled by updating the Helm chart, enabling rapid adoption of new capabilities.
+
+* **Inspectable Cluster Assembly:** The Helm chart details clearly outline how the cluster is assembled, providing transparency and diagnostic capabilities.
+
+* **Documented Configuration Options:** Comprehensive documentation is provided as part of the JSON schema, ensuring easy understanding and reference.
+
+* **Happa UI-Based Cluster Creation:** The Happa UI streamlines cluster creation from the JSON schema, enhancing user experience.
+
+### Configurable Registries with Credentials and Mirrors
+
+CAPI enables the flexibility to specify custom registries with credentials and mirrors, allowing customers to utilize their preferred container repositories. This flexibility enhances the portability and adaptability of customer environments.
+
+### Removal of Cluster Management Operators
+
+The legacy cluster management operators have been removed due to several limitations and drawbacks:
+
+* **Incompatibility with Multi-Version CAPI:** Upstream CAPI doesn't support running multiple versions of operators on the same management cluster, creating operational challenges.
+
+* **Lack of On-Demand Updates:** Operator bug fixes required a full release and upgrade cycle for both GS and customers, hindering timely remediation.
+
+CAPI addresses these limitations with a new approach:
+
+* **Cross-Version Compatibility:** New operator versions can manage all previously provisioned clusters, ensuring ongoing support for existing deployments.
+
+* **On-Demand Upgrades:** Cluster upgrades are triggered explicitly per cluster, preventing unplanned or unintended upgrades.
+
+* **Enhanced Testing for Operators:** Thorough testing ensures the quality and reliability of the operators.
```

Comment:
```suggestion
* **Enhanced Testing for Operators:** Thorough testing ensures the quality and reliability of the operators.
```
IMO we need to refresh since this one sounds like before we did not test anything